Sources/SimpleLogger/Logger.swift

```diff
@@ -134,6 +140,13 @@ public struct Logger<Topic: LoggerTopic> {
         return logDirectory
     }
 
+    private func getUniqueDeviceId() -> UUID {
+        if let deviceId = UIDevice.current.identifierForVendor {
+            return deviceId
+        }
+        return UUID()
```

Comment:
This would be a good opportunity to use `guard let`. I am not sure if `UUID()` should be used in the case the `deviceId` can't be received. Maybe UUID should be Optional in this case. UUID would deliver always a new unique identifier, which is not similar to the previous one.

Reply:
Addressed this issue in my last commit. Not sure if the error handling is ideal already though

Reply:
This should work. Regarding the [Documentation](https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor), it is only `NIL`, if the device is unlocked.